components/brave_wallet_ui/panel/async/wallet_panel_async_handler.ts

```diff
@@ -387,7 +387,7 @@ handler.on(PanelActions.signMessageHardware.getType(), async (store, messageData
   }
   const payload: SignMessageHardwareProcessedPayload =
     signed.success ? { success: signed.success, id: messageData.id, signature: signed.payload }
-                   : { success: signed.success, id: messageData.id, error: signed.error }
+      : { success: signed.success, id: messageData.id, error: signed.error }
```

Comment:
Let's follow this style.

```
condition
  ? then
  : else
```

Reply:
Oh weird! My code formatter must have picked up on this when I saved my changes in this file.
No related to this PR, but fixed the code styling any way. 😎